editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -2,26 +2,116 @@
 
 import React from 'react'
 import { jsx } from '@emotion/react'
+import styled from '@emotion/styled'
 
 import TailWindList from '../../../core/third-party/tailwind-all-classnames.json'
-import WindowedSelect from 'react-windowed-select'
+import WindowedSelect, {
+  components,
+  createFilter,
+  IndicatorProps,
+  MultiValueProps,
+  ValueContainerProps,
+} from 'react-windowed-select'
+import type { StylesConfig } from 'react-select'
+import chroma from 'chroma-js'
 
 import * as EditorActions from '../../editor/actions/action-creators'
 import { betterReactMemo } from '../../../uuiui-deps'
-import { useColorTheme, UtopiaTheme } from '../../../uuiui'
+import { useColorTheme } from '../../../uuiui'
 import { useEditorState } from '../../editor/store/store-hook'
 import { MetadataUtils } from '../../../core/model/element-metadata-utils'
 import * as EP from '../../../core/shared/element-path'
 import * as PP from '../../../core/shared/property-path'
 import { jsxAttributeValue } from '../../../core/shared/element-template'
 import { emptyComments } from '../../../core/workers/parser-printer/parser-printer-comments'
 
+interface TailWindOption {
+  label: string
+  value: string
+  categories?: string[]
+}
+
 const TailWindOptions = TailWindList.classNames.map((className, index) => ({
   label: className,
   value: className,
 }))
 
+const DropdownIndicator = (props: IndicatorProps<TailWindOption, true>) => (
+  <components.DropdownIndicator {...props}>
+    <span> ↓ </span>
+  </components.DropdownIndicator>
+)
+
+const ClearIndicator = () => null
+const IndicatorSeparator = () => null
+
+const NoOptionsMessage = (props: any) => <span {...props}>No other classes available</span>
```

Comment:
Is this what we display when there are no results? If so, I think this should read "No results found"

Reply:
fixed